test/jasmine/tests/hover_label_test.js

```diff
@@ -5377,36 +5377,134 @@ describe('hovermode: (x|y)unified', function() {
             .then(function(gd) {
                 _hover(gd, { xpx: 40, ypx: 200 });
                 assertLabel({title: 'Jan', items: [
-                    'bar : (Jan 1, 2000, 1)',
-                    'start : 1',
-                    'end : 1'
+                    'start : 1'
                 ]});
 
                 _hover(gd, { xpx: 100, ypx: 200 });
+                assertLabel({title: 'Jan 1, 2000', items: [
+                    'bar : 1'
+                ]});
+
+                _hover(gd, { xpx: 210, ypx: 200 });
                 assertLabel({title: 'Jan', items: [
                     'bar : (Jan 1, 2000, 1)',
```

Comment:
Line [5348](https://github.com/plotly/plotly.js/pull/5846/files#diff-cfd23075dfa021e046f24be2b4f2cf2ee1a8e86aeec566e4d66dea1c09218140R5348) has a typo that's making this test confusing (`xhoverfrmat`)

Reply:
Good eye. Addressed in https://github.com/plotly/plotly.js/pull/5846/commits/7c37e982f164cc6951954fbbb9c7adb46b5f5385.